.github/workflows/push-incremental-results.yml

```
@@ -137,4 +137,6 @@ jobs:
           git commit -m "chore(stryker): update incremental results"
 
       - name: Push Incremental Reports 🚀
-        run: git push origin ${{ github.event.inputs.target-branch }}
\ No newline at end of file
+        run: |
+          git remote set-url origin https://x-access-token:${{ secrets.PAT_PUSH_TOKEN }}@github.com/${{ github.repository }}
+          git push origin ${{ github.event.inputs.target-branch }}
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)